task-list/app/models/person.rb

```diff
@@ -0,0 +1,10 @@
+class Person < ActiveRecord::Base
+  has_many :tasks
+
+  def count_incomplete
```

Comment:
This is a pretty good use of a method on a model class!

For optimal usage, this should be located in the `Task` model as a class method, and instead of `self.tasks` you would say `Task.all`.
Even though the code would then be returning the number of incomplete tasks for all people, ActiveRecord is smart enough to know that if you say `@person.tasks.count_incomplete` you're only looking for the number that are associated with `@person`.